Documentation website for GCBM.Colombia (#9)
moja-global/GCBM.Colombia: docs/about_colombia/index.rst

About Colombia
==============

(Source - Wikipedia)

Colombia is a country in South America with an insular region in North America. 

> It is bordered by the Caribbean Sea to the north, Venezuela to the east, 

Reviewer: ```suggestion
The Caribbean Sea borders it to the north, Venezuela to the east,  
```

Brazil to the southeast, Ecuador and Peru to the south, the Pacific Ocean to the west and 
Panama to the northwest. Colombia comprises 32 departments and the Capital District of Bogotá, 
the country's largest city. It covers an area of 1,141,748 square kilometers (440,831 sq mi), with a population of 50 million.

The geography of Colombia is characterized by its six main natural regions that 

> present their own unique characteristics, from the Andes mountain range region 

Reviewer: ```suggestion
present their unique characteristics from the Andes mountain range region 
```

shared with Ecuador and Venezuela; the Pacific Coastal region shared with Panama and Ecuador; 
the Caribbean coastal region shared with Venezuela and Panama; the Llanos (plains) 
shared with Venezuela; the Amazon rainforest region shared with Venezuela, Brazil, 

> Peru and Ecuador; to the insular area, comprising islands in both the Atlantic and Pacific oceans.

Reviewer: ```suggestion
Peru and Ecuador; to the insular area, comprising islands in the Atlantic and Pacific oceans.
```

> The climate of Colombia is characterized for being tropical presenting variations within six natural regions 

Reviewer: ```suggestion
The climate of Colombia is characterized as being tropical, presenting variations within six natural regions
```

and depending on the altitude, temperature, humidity, winds and rainfall. 

> Colombia has a diverse range of climate zones, including tropical rainforests, savannas, steppes, deserts and mountain climates.

Reviewer: ```suggestion
Colombia has diverse climate zones, including tropical rainforests, savannas, steppes, deserts and mountain climates.
```
